chia/types/eligible_coin_spends.py

```diff
@@ -332,17 +331,18 @@ async def process_fast_forward_spends(
             coin_spends=new_coin_spends, aggregated_signature=mempool_item.spend_bundle.aggregated_signature
         )
         # We need to run the new spend bundle to make sure it remains valid
-        generator = simple_solution_generator(new_sb)
-        new_npc_result = get_name_puzzle_conditions(
-            generator=generator,
-            max_cost=mempool_item.conds.cost,
-            mempool_mode=True,
-            height=height,
-            constants=constants,
-        )
-        if new_npc_result.error is not None:
-            raise ValueError("Mempool item became invalid after singleton fast forward.")
-        assert new_npc_result.conds is not None
+        # generator = simple_solution_generator(new_sb)
+        assert mempool_item.conds is not None
+        try:
+            new_sbc_result = get_conditions_from_spendbundle(
```

Comment:
I would name this `new_conditions` or `new_conds` to fit their use below, better.

Reply:
Done